dataland-frontend/src/components/resources/frameworkDataSearch/euTaxonomy/EuTaxonomyForNonFinancialsPanel.vue

```diff
@@ -47,7 +48,17 @@ export default defineComponent({
       convertedDataAndMetaInfo: [] as Array<DataAndMetaInformationEuTaxonomyForNonFinancialsViewModel>,
       euTaxonomyForNonFinancialsModalColumnHeaders,
       euTaxonomyForNonFinancialsDisplayDataModel,
-      namesOfFieldsToFormatAsPercentages: ["relativeShareInPercent", "totalEnablingShare", "totalTransitionalShare"],
+      namesOfFieldsToFormatAsPercentages: [
+        "relativeShareInPercent",
```

Comment:
Works for now, will be fixed in the upload branch though.

Reply:
Is gone now, since I took over the same solution from the upload branch.

Reply:
perfect